packages/styletron-engine-atomic/src/css.ts

```diff
@@ -13,7 +11,7 @@ export function atomicSelector(id: string, pseudo: string): string {
   return selector;
 }
 
-export function keyframesToBlock(keyframes: {[string]: Object}): string {
+export function keyframesToBlock(keyframes: {[x: string]: any}): string {
```

Comment:
probably, was not really focusing on improving types(this is exactly what it was in flow types)

I think it might better to have type improvements as separate from this, as an option starting from enabling strict mode which will highlight where to start :)

Reply:
Ah I didn't realize that this fellback to `any` in flow. Thought it would at least loosely check https://flow.org/try/#0C4TwDgpgBA8lC8sBGArCBjYBuAUD9A9gHYDOwUAhgFywJQCMWU+xZUSNciA5Bd7oVLl0nOkQCuAG0lYgA